articles/virtual-machines/linux/capture-image.md

```diff
@@ -89,6 +89,8 @@ Use the Azure CLI to mark the VM as generalized and capture the image. In the fo
    > The image is created in the same resource group as your source VM. You can create VMs in any resource group within your subscription from this image. From a management perspective, you may wish to create a specific resource group for your VM resources and images.
    >
    > If you would like to store your image in zone-resilient storage, you need to create it in a region that supports [availability zones](../../availability-zones/az-overview.md) and include the `--zone-resilient true` parameter.
+   
+This command will return JSON describing the VM image.  Save this output for later reference.  At this point, the VM from which the image was created will have been shut down by Azure in the process of executing the preceding commands. 

```

Comment:
The VM is actually shut down during the deallocation. If you want to clarify when the VM stops running, I would add it to line 70, but use language that is a little more user friendly ("The VM is shut down during deallocation.") Marking the VM as `generalized` is usually the point of no return - after that point it can't\shouldn't be restarted. Other topics about generalized images talk about that and it could be added here too (I'll add it if you don't).

Reply:
"Executing" is avoided because it can be difficult for localization.